views/files-storage/files.hbs

```diff
@@ -0,0 +1,30 @@
+{{#extend "lib/loggedin"}}
+    {{#content "styles" mode="append"}}
+        <link rel="stylesheet" href="/styles/files/files.css" />
+    {{/content}}
+
+    {{#content "scripts" mode="append"}}
```

Comment:
Not sure with the css files, but for js files i think it should work to put it into the embedded hbs file and it should work. For html + js it self it is no problem to but js file loads into body, or we can do it to fileStorageImports.hbs like imprts.hbs file.